netbox/dcim/tests/test_models.py

```diff
@@ -439,6 +453,37 @@ def test_cable_cannot_terminate_to_an_existing_connection(self):
         with self.assertRaises(ValidationError):
             cable.clean()
 
+    def test_singlepos_rearport_connections(self):
+        """
+        A RearPort with one position can be connected to anything as it is just a
+        cable extender.
+        """
+        # Connecting a single-position RearPort to a multi-position RearPort is ok
+        Cable(termination_a=self.rear_port1, termination_b=self.rear_port2).full_clean()
+
+        # Connecting a single-position RearPort to an Interface is ok
+        Cable(termination_a=self.rear_port1, termination_b=self.interface3).full_clean()
+
+    def test_multipos_rearport_connections(self):
```

Comment:
Needs a topology diagram like the other tests. Name should be `test_connection_via_multi_position_rearport`.